circuit-prover/src/config.rs

```diff
@@ -87,7 +87,7 @@ where
     let challenge_mmcs = ChallengeMmcs::<F, P, CD>::new(val_mmcs.clone());
 
     let dft = Dft::<F>::default();
-    let fri_params = create_test_fri_params::<ChallengeMmcs<F, P, CD>>(challenge_mmcs, 0);
+    let fri_params = create_benchmark_fri_params::<ChallengeMmcs<F, P, CD>>(challenge_mmcs);
```

Comment:
unrelated, but the test config only has 2 queries, I think it's best to use a regular config from the start. Doesn't affect initial proving anyway